File: src/exercises/pov/ocaml/template.ml
type tree = { value : string; children : tree list }

let from_pov (target : string) (tr : tree) : tree option =
  (* Your code here *)
  Some tr

let path_to (from_node : string) (to_node : string) (tr : tree) : string list option =
  (* Your code here *)
  Some []

File: src/exercises/pov/python/template.py
class Tree:
    def __init__(self, label: str, children: list = None):
        self.label = label
        self.children = children or []

    def from_pov(self, from_node: str):
        # Your code here
        return self

    def path_to(self, from_node: str, to_node: str) -> list[str]:
        # Your code here
        return []
